l10n/data/uk.json

```
@@ -0,0 +1,550 @@
+{
+    "bot_start_text": [
+        "👋🏼 Привітики, {}!",
+        "Цей бот використовується для отримання інформації про стан серверів та перевірки онлайну Counter-Strike: Global Offensive та багато іншого.",
+        "",
+        "Для деталей використовуйте /help."
+    ],
+    "bot_help_text": [
+        "[‎](https://telegra.ph/Detalnij-opis-08-14)ℹ️ Цей бот розроблений каналом @INCS2. Його код можна знайти за цим посиланням [GitHub](https://github.com/aquaismissing/INCS2bot).",
```

```suggestion
        "[‎](https://telegra.ph/Detalnij-opis-08-14)ℹ️ Цей бот розроблений каналом @INCS2. Програмний код можна знайти на [GitHub](https://github.com/aquaismissing/INCS2bot).",
```
Для простоты и укрощения будет лучше так написать

---

👍